index.html

```diff
@@ -148,40 +148,16 @@ <h1 class="lets">My Recent Works</h1>
         </div><img src="img/circle-6.png" class="circle-6" alt="circle">
         </section>
 
-    <section class="contact-me">
+    <footer class="contact-me">
 <p class="lets">Let’s chat!</p>
 <p class="contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-<form method="POST" class ="contactForm">
-    <div class="row">
-    <div class="small-fe">
-        <div>
-    <input type="text" class="form-control" name="firstname" id="firstname" placeholder="First Name">
-    </div>
-    </div>
-    <div class="small-fe">
-        <div>
-    <input type="email" class="form-control" name="lastname" id="lastname" placeholder="Last Name">
-    </div>
-    </div>
-
-    <div class="big-fe">
-    <div>
-    <input type="text" class="form-control" name="email" id="email" placeholder="Email">
-    </div>
-    </div>
-    <div class="big-fe">
-    <div>
-    <textarea name="message" class="form-massage" id="message" cols="30" rows="7" placeholder="Write Your Massage Here"></textarea>
-    </div>
-    </div>
-    <div class="big-fe">
-    <div>
+<form method="POST" class ="contactForm" action="https://formspree.io/f/mqkwgokn">
```

Comment:
- [ ] It seems that your website is only responsible for the screen sizes > 560px, this is how your form is looking like on 480px 👇 
![image](https://user-images.githubusercontent.com/55465133/130667148-048d8a7f-2812-4317-a12b-26a024c68f15.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130667734-f72b7350-275a-43eb-bbd5-59275b06cd96.png)

As you can see your form is not well centered, Kindly let's play around with margins, padding, and alignments to center the form and thus, make it look like the one in the Figma design 😉 

- [ ] You can also give to the `textarea` the same `font-family` as other inputs 👍 
- [ ] Kindly let's give some padding-left to all the `<input>` and `<textarea>`, because for now the content of the input is stuck to the left.